subsys/emul/espi/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Configuration options for eSPI emulators
+
+# Copyright 2020 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config EMUL_ESPI_HOST
+	bool "Emulate an eSPI host"
+	help
+	  This is an emulator of the eSPI host.
```

Comment:
what does it do? What features does it have?

Reply:
Done.